requirements-dev-only-frozen.txt

```diff
@@ -50,7 +50,7 @@ pyparsing==3.0.9
 pyrsistent==0.18.1
 pytest==6.1.0
 pytest-cov==2.11.1
-pytest-django==3.10.0
+pytest-django==4.4.0
```

Comment:
for posterity, see https://github.com/getsentry/sentry/pull/35732

Reply:
yeah I mostly did something trivial so I could make sure my baseclass change didn't break `getsentry` -- and I had to fix 1 test there heh